ReactFrontEnd/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "wallstreetbets-visualizer",
+    "version": "0.1.0",
+    "private": false,
+    "main": "dist/index.js",
+    "description": "The frontend website for displaying data for WallStreetBets Analytics Tool",
+    "scripts": {
+        "build": "rimraf ./dist*/ && cross-env webpack --config webpack.config.js --mode production",
+        "dev": "cross-env NODE_ENV=development webpack serve --mode development"
+    },
+    "devDependencies": {
+        "@babel/core": "^7.12.10",
+        "@babel/plugin-transform-runtime": "^7.12.10",
+        "@babel/preset-env": "^7.12.11",
+        "@babel/preset-react": "^7.12.10",
+        "babel-loader": "^8.2.2",
+        "cross-env": "^7.0.3",
+        "html-webpack-plugin": "^4.5.1",
+        "react": "^17.0.1",
+        "react-dom": "^17.0.1",
+        "rimraf": "^3.0.2",
+        "sass": "^1.32.6",
+        "sass-loader": "^11.0.1",
+        "style-loader": "^2.0.0",
+        "url-loader": "^4.1.1",
+        "webpack": "^5.11.1",
+        "webpack-cli": "^4.3.1",
+        "webpack-dev-server": "^3.11.2"
+    },
+    "dependencies": {
+        "css-loader": "^5.0.2"
```

Comment:
I vote for scss here instead of css.

Reply:
so if you look in the webpack.config.js file on lines 32 - 40, this merge request is actually works with all 3: scss, css, and sass files. there is an issue with the different loaders on windows machines in webpack 5 and sass, and this repo has the fix installed as a fall back as the 3rd loader (sass-loader).

as for the quick demo, i just used a plain old .css file for initial pull request purposes but if you'd prefer to use scss or sass we can definitely enforce that standard going forward.

Reply:
You get my vote as well, also before we go off merging this project is certainly going to consist of multiple repos so better naming convention for this repo might be good.